src/assets/js/main.js

```diff
@@ -670,77 +670,72 @@ jQuery(document).ready(function ($) {
 
     var $container = this;
 
-    $.get('https://blog.mozilla.org/addons/feed/', function (data) {
-      var $xml = $(data); // DOMPurify.sanitize(data) doesn't work here because data is not html
-      var i = settings.num;
-      $xml.find('item').each(function () {
-        if (i--) {
-          var $this = $(this),
-            item = {
-              title: DOMPurify.sanitize($this.find('title').text()),
-              link: DOMPurify.sanitize($this.find('link').text()),
-              description: DOMPurify.sanitize($this.find('description').text()),
-              pubDate: DOMPurify.sanitize($this.find('pubDate').text()),
-            };
-
-          var newDate = new Date(item.pubDate);
-          var d = newDate.getDate();
-          var m = newDate.getMonth();
-          m += 1; // JavaScript months are 0-11
-          var y = newDate.getFullYear();
-          var formattedDate = y + '/' + pad(m, 2) + '/' + pad(d, 2);
-
-          var $description_elements = $(item.description);
-
-          var $cell = $(settings.container);
-          var $cell_content = $cell.find('.block-link');
-
-          $cell_content
-            .append(
+    $.get(
+      'https://blog.mozilla.org/addons/feed/',
+      function (doc) {
```

Comment:
Not needed. Passing "xml" ensures that it's parsed as a document.

Reply:
Yeah I know understood that change, I was thinking more along the lines of what if that got removed.

Reply:
I've added the comment. To make the code more readable (i.e. the relation between 'xml', `$.get` and the comment more explicit), I moved the handler to a new method so that `$.get` and `'xml'` are on one line, next to the comment.

(a nice side effect is that the diff has become smaller again on Github, not hindered by prettier)

Reply:
Yep, that's perfect - many thanks!